lib/src/pages/conversation.dart

```diff
@@ -82,76 +84,95 @@ class ConversationPage extends ReactiveWidget<ConversationViewModel> {
   }
 
   @override
-  Widget build(BuildContext context, ConversationViewModel model) => Scaffold(
-    appBar: AppBar(
-      // If the keyboard is open, it will close and ruin the hero animation. 
-      // This workaround overrides the back button to close the keyboard, wait
-      // a very small delay, *then* go back. 
-      // See: https://github.com/flutter/flutter/issues/86089
-      leading: IconButton(
-        icon: const Icon(Icons.arrow_back),
-        onPressed: () async {
-          FocusManager.instance.primaryFocus?.unfocus();
-          await Future<void>.delayed(const Duration(milliseconds: 100));
-          if (!context.mounted) return;
-          context.pop();
-        },
+  Widget build(BuildContext context, ConversationViewModel model) {
+    Future<void> showReportForm() async => showDialog<void>(
+      context: context,
+      builder: (BuildContext context) => ReportDialogue(
+        itemType: ItemType.conversation, 
+        itemID: id.id,
       ),
-      actions: [
-        if (!model.conversation.isSeller) TextButton(
-          style: TextButton.styleFrom(foregroundColor: context.colorScheme.onPrimary),
-          onPressed: model.openSellerProfile, 
-          child: const Text("View profile"),
+    );
```

Comment:
Try breaking this out as a separate method that accepts a `BuildContext`, and leave the `build()` to be a simple `=>` function